test_objects.py

```diff
@@ -89,11 +89,18 @@ def test_object_relationships():
     dessert3 = Dessert(price=13)
     cake1 = Cake(kind='sponge')
     cake2 = Cake(kind='birthday')
-    my_desserts = [dessert1, dessert2, dessert3, cake1, cake2]
+    not_a_cake1 = 15
```

Comment:
it's nice to keep your test logic a little separate, so start your new test after the provided code, rather than mixing it up :)